src/components/fields/select-field/README.md

```diff
@@ -0,0 +1,83 @@
+| `aria-label`            | `string`           |    -     | -                                  | -       | Aria label (for assistive tech)                                                                                                                                                                                                                                                             |
+| `aria-labelledby`       | `string`           |    -     | -                                  | -       | HTML ID of an element that should be used as the label (for assistive tech)                                                                                                                                                                                                                 |
+| `isAutofocussed`        | `bool`             |    -     | -                                  | -       | Focus the control when it is mounted                                                                                                                                                                                                                                                        |
+| `backspaceRemovesValue` | `bool`             |    -     | -                                  | `true`  | Remove the currently focused option when the user presses backspace                                                                                                                                                                                                                         |
+| `components`            | `objectOf(func)`   |    -     | -                                  | -       | Map of components to overwrite the default ones, see [what components you can override](https://react-select.com/components)                                                                                                                                                                |
+| `filterOption`          | `func`             |    -     | -                                  | -       | Custom method to filter whether an option should be displayed in the menu                                                                                                                                                                                                                   |
+| `inputId`               | `string`           |    -     | -                                  | -       | The id of the search input                                                                                                                                                                                                                                                                  |
```

Comment:
what would this be used for? maybe we can add the use case here?

Reply:
Sorry, this is outdated by now. `react-select` offers `id` (for the container) and `inputId` (for the input element).

As we want to be able to use labels to focus the search input we need to provide a way to set the `id` of the `input`. To avoid confusion we offer `id` (the id of the input)  and `containerId` (the id of the container) as props. They are mapped to `id` and `inputId` internally.

```
<Select
  id={props.containerId}
  inputId={props.id}
```

Reply:
I force-pushed an update to the docs.